Trust and Compliance: Fiscal provisions (#105)
pretix/pretix-docs: docs/trust/fiscal/germany.md

# Germany

{% include "warning-tax-law.md" %}

## Value Added Tax (VAT)

When selling goods and services in Germany, VAT ("Umsatzsteuer") may apply.

> All German VAT rates can be configured in pretix using [tax rules](../../guides/taxes.md), including more complex tax cases (e.g. intra-EU reverse charge transactions).

Reviewer: ```suggestion
You can configure all German VAT rates in pretix using [tax rules](../../guides/taxes.md), including more complex tax cases (e.g. intra-EU reverse charge transactions).
```

## Invoicing

### Invoice information

> German requirements on information contained on invoices are defined in [§ 14 (4) UStG](https://www.gesetze-im-internet.de/ustg_1980/__14.html).

Reviewer: ```suggestion
German requirements on information included in invoices are defined in [§ 14 (4) UStG](https://www.gesetze-im-internet.de/ustg_1980/__14.html).
```

### E-invoicing

Germany is currently implementing an electronic invoicing mandate.
All invoices sent in a business-to-business (B2B) relationship must be issued in a structured electronic format.
An exception is made for invoices below €250 and for small businesses (those that fall under [§ 19 UStG](https://www.gesetze-im-internet.de/ustg_1980/__19.html)).

Up to the end of 2026 (2027 for smaller businesses), sending paper or PDF invoices is still allowed, although PDF invoices may require the consent of the receiver.

Any type of electronic invoice is allowed as long as it is compliant with European norm EN16931 and accepted by the receiver.
The most common formats are [ZUGFeRD](https://de.wikipedia.org/wiki/ZUGFeRD) and [XRechnung](https://de.wikipedia.org/wiki/XRechnung).

pretix supports generating ZUGFeRD invoices compliant with EN16931 using the [ZUGFeRD plugin](https://marketplace.pretix.eu/products/zugferd/).
ZUGFeRD invoices are hybrid documents with a PDF part and an XML part.
This way, they can be processed as electronic invoices by German B2B customers, and as regular PDF invoices by everyone else.

> Additional guidance is available in German on [our blog](https://pretix.eu/about/de/blog/20241218-e-rechnung-starten/).

Reviewer: ```suggestion
Additional guidance is available in our German blog article [Durchstarten mit der E-Rechnung](https://pretix.eu/about/de/blog/20241218-e-rechnung-starten/).
```

## Audit readiness

### Procedure documentation

All German companies using electronic systems in their financial processes are required to have internal documentation of their processes ([10.1 GoBD](https://ao.bundesfinanzministerium.de/ao/2023/Anhaenge/BMF-Schreiben-und-gleichlautende-Laendererlasse/Anhang-64/inhalt.html)).

> This documentation should include, for example, how you use pretix, who has access to pretix, and how data is transferred from pretix to your accounting systems.

Reviewer: ```suggestion
This documentation should include, for example, how you use pretix, who has access to your pretix account or pretix server, and how you transfer data from pretix to your accounting systems.
```

To include technical details about pretix in your procedure documentation, you can use our [Annex for the Procedure Documentation](https://download.pretix.eu/vd.pdf).

### Logging of changes

> All changes to configurations or transaction data within pretix is logged.

Reviewer: ```suggestion
pretix logs all changes to configurations or transaction data. 
```

This logging includes the user, time, and type of change made in the system.

> The software does not include functionality to modify or delete these logs (other than by applying pseudonymization of personal information through our privacy tools).

Reviewer: ```suggestion
pretix does not allow you to modify or delete these logs (other than by applying pseudonymization of personal information through our privacy tools).
```

> Orders can only be deleted from the system (unless they were made during test mode).

Reviewer: ```suggestion
You can only delete orders deleted from the system (unless they were made during test mode).
```
What system? Oberflächlich scheint das dem vorigen Satz zu widersprechen

We therefore consider pretix to be audit-proof as required by law (e.g. [§ 146 (4) AO](https://www.gesetze-im-internet.de/ao_1977/__146.html), [§ 293 (3) HGB](https://www.gesetze-im-internet.de/hgb/__239.html)).

### Archiving

> All invoices and receipts need to be archived digitally for at least 8 years ([§ 257 HGB](https://www.gesetze-im-internet.de/hgb/__257.html), [§ 147 AO](https://www.gesetze-im-internet.de/ao_1977/__147.html), [§ 14b UStG](https://www.gesetze-im-internet.de/ustg_1980/__14b.html)).

Reviewer: ```suggestion
Archive all invoices and receipts digitally for at least 8 years ([§ 257 HGB](https://www.gesetze-im-internet.de/hgb/__257.html), [§ 147 AO](https://www.gesetze-im-internet.de/ao_1977/__147.html), [§ 14b UStG](https://www.gesetze-im-internet.de/ustg_1980/__14b.html)).
```

Reviewer: Ja, das ginge. Alternativ auch: 

```suggestion
The legislation requires you to archive all invoices and receipts need to be archived digitally for at least 8 years ([§ 257 HGB](https://www.gesetze-im-internet.de/hgb/__257.html), [§ 147 AO](https://www.gesetze-im-internet.de/ao_1977/__147.html), [§ 14b UStG](https://www.gesetze-im-internet.de/ustg_1980/__14b.html)).
```

While pretix does not delete old invoices, it is not a suitable system to provide sufficient guarantees about long-term archival.
We strongly recommend regularly exporting all created invoices and storing them in an archival system.

> Additional material required to understand your accounting is also required to be archived (see e.g. [1.3, 5.2 GoBD](https://ao.bundesfinanzministerium.de/ao/2023/Anhaenge/BMF-Schreiben-und-gleichlautende-Laendererlasse/Anhang-64/inhalt.html)).

Reviewer: ```suggestion
You also have to archive additional material that is required for understanding your accounting (see e.g. [1.3, 5.2 GoBD](https://ao.bundesfinanzministerium.de/ao/2023/Anhaenge/BMF-Schreiben-und-gleichlautende-Laendererlasse/Anhang-64/inhalt.html)).
```

This includes at least order data, payment data, and your procedure documentation.

> We recommend keeping this data in pretix for easy exporting and processing, but also exporting an offline copy in regular intervals (see also [9.4 GoBD](https://ao.bundesfinanzministerium.de/ao/2023/Anhaenge/BMF-Schreiben-und-gleichlautende-Laendererlasse/Anhang-64/inhalt.html)).

Reviewer: ```suggestion
We recommend keeping this data in pretix for easy exporting and processing, but also exporting an offline copy regularly (see also [9.4 GoBD](https://ao.bundesfinanzministerium.de/ao/2023/Anhaenge/BMF-Schreiben-und-gleichlautende-Laendererlasse/Anhang-64/inhalt.html)).
```

## pretixPOS

pretixPOS can be used in compliance with German laws on cash registers ([§ 146a AO](https://www.gesetze-im-internet.de/ao_1977/__146a.html), [KassenSichV](https://www.gesetze-im-internet.de/kassensichv/BJNR351500017.html)).

Use of a cash register is not mandatory in Germany.

> However, if a cash register is used, it must comply with the following requirements:

Reviewer: ```suggestion
If you use a cash register, take the following steps to ensure that you are complying with legal requirements:
```

> - Usage of a [TSE](../../guides/pretixPOS/tse.md), a specialized device that ensures that no modification of recorded data can go unnoticed.

Reviewer: ```suggestion
- Use a so-called [technische Sicherheitseinrichtung](../../guides/pretixPOS/tse.md) (TSE), a specialized device that ensures that no modification of recorded data can go unnoticed.
```

>   pretixPOS is compatible with TSEs of multiple vendors.

Reviewer: ```suggestion
  pretixPOS is compatible with TSEs from multiple vendors.
```

> - Registration of the cash register with the tax authorities.

Reviewer: ```suggestion
- Register all cash registers with the tax authorities.
```

  pretix includes [utilities to simplify registration](../../guides/pretixPOS/register.md).

> - Issuing of receipts with all information required by [§ 6 KassenSichV](https://www.gesetze-im-internet.de/kassensichv/BJNR351500017.html) and [§ 14 (4) UStG](https://www.gesetze-im-internet.de/ustg_1980/__14.html).

Reviewer: ```suggestion
- Issue receipts with all information required by [§ 6 KassenSichV](https://www.gesetze-im-internet.de/kassensichv/BJNR351500017.html) and [§ 14 (4) UStG](https://www.gesetze-im-internet.de/ustg_1980/__14.html).
```

  pretixPOS allows issuing compliant receipts using a receipt printer or in digital form.

> - Compatibility with a defined export format.

Reviewer: ```suggestion
- Export data in an officially defined format. 
```

>   pretixPOS is able to export data in the [DSFinV-K](https://www.bzst.de/DE/Unternehmen/Aussenpruefungen/DigitaleSchnittstelleFinV/digitaleschnittstellefinv_node.html) format.

Reviewer: ```suggestion
  pretixPOS can export data in the [DSFinV-K](https://www.bzst.de/DE/Unternehmen/Aussenpruefungen/DigitaleSchnittstelleFinV/digitaleschnittstellefinv_node.html) format.
```

> We are members of the [German association of cash register manufacturers](https://dfka.net/) to always stay updated with the latest changes in cash register regulation.

Reviewer: ```suggestion
rami.io GmbH is a member of the [German association of cash register manufacturers](https://dfka.net/). 
This means that we always stay up-to-date with changes in cash register regulation.
```
